.for_maintainers/release/RELEASE_NOTES_TEMPLATE.md

```diff
@@ -0,0 +1,167 @@
+    - [Upgrading Wallaroo in Vagrant](#upgrading-wallaroo-in-vagrant)
+5. [CHANGELOG](#changelog)
+
+<a name="added"></a>
+## Added
+
+<a name="example"></a>
+### Example
+
+<a name="changed"></a>
+## Changed
+
+<a name="fixed"></a>
+## Fixed
+
+<a name="upgrading-wallaroo"></a>
+## Upgrading Wallaroo
+
+In all cases below, if you run into issues, please reach out to us! We’re available on [twitter](https://twitter.com/wallaroolabs), [IRC](https://webchat.freenode.net/?channels=#wallaroo), [Github](https://github.com/WallarooLabs/wallaroo), [by email](mailto:hello@wallaroolabs.com), or [our mailing list](https://groups.io/g/wallaroo).
+We love questions!
+
+**If you have made no changes to Wallaroo or Pony since installation, your best bet will be to delete your Wallaroo installation and start from scratch, following the [instructions] (https://docs.wallaroolabs.com/book/getting-started/choosing-an-installation-option.html) of your choice.**
+
+Below are instructions for [Upgrading Wallaroo when compiled from source](#upgrading-wallaroo-when-compiled-from-source), [Upgrading Wallaroo in Docker](#upgrading-wallaroo-in-docker), [Upgrading Wallaroo in Vagrant](#upgrading-wallaroo-in-vagrant), and [Upgrading Wallaroo via Wallaroo Up](#upgrading-wallaroo-up).
+
+<a name="upgrading-wallaroo-when-compiled-from-source"></a>
+### Upgrading Wallaroo when compiled from source
+
+These instructions are for Ubuntu Linux. It's assumed that if you are using a different operating system then you are able to translate these instructions to your OS of choice.
+
+#### Upgrading ponyc to x.x.x
+
+`ponyc` can be upgraded with the following command:
+
+```bash
+sudo apt-get install --only-upgrade ponyc=x.x.x
+```
+
+Verify you are now on the correct version of ponyc by running:
+
+```bash
+ponyc --version
+```
+
+You should get the following output:
+
+```bash
+x.x.x [release]
+```
+
+#### How to Upgrade Wallaroo
+
+Once you're on the latest ponyc and pony stable, you're ready to switch over to Wallaroo x.x.x.
+
+We recommend moving your current Wallaroo directory and starting with a fresh clone of the latest release. If you have made prior changes to the Wallaroo code, you’ll need to re-implement those changes. To get a fresh clone, assuming that you cloned the repository to the directory we recommended in setup, you’ll need to run the following:
+
+```bash
+cd ~/wallaroo-tutorial/
+```
+
+To get a new copy of the Wallaroo repository, run the following commands:
+
+```bash
+cd ~/wallaroo-tutorial/
+curl -L -o wallaroo-x.x.x.tar.gz 'https://wallaroo-labs.bintray.com/wallaroolabs-ftp/wallaroo/x.x.x/wallaroo-x.x.x.tar.gz'
+mkdir wallaroo-x.x.x
+tar -C wallaroo-x.x.x --strip-components=1 -xzf wallaroo-x.x.x.tar.gz
+rm wallaroo-x.x.x.tar.gz
+cd wallaroo-x.x.x
+```
+
+You can then run the following commands to build the necessary tools to continue developing using Wallaroo x.x.x:
+
+```bash
+cd ~/wallaroo-tutorial/wallaroo
```

Comment:
This should refer to the `wallaroo-x.x.x` directory instead of the `wallaroo` directory.